ingest/expression_files/mtx.py

```diff
@@ -157,10 +157,24 @@ def check_bundle(barcodes, genes, mtx_dimensions):
 
         if (actual_barcodes == expected_barcodes) and (actual_genes == expected_genes):
             return True
+        elif (actual_barcodes == expected_genes) and (
+            actual_genes == expected_barcodes
+        ):
```

Comment:
Having the full clause on one line would make it easier to compare to the other clauses.

```suggestion
        elif (actual_barcodes == expected_genes) and (actual_genes == expected_barcodes):
```

Reply:
Unfortunately that formatting is performed by `Black` which prides itself on being "uncompromising" (ie. no facility for applying formatting exceptions)